tapis-project/tapis-ui
# task/TUI-246 -- exec system options

## Overview:

Add execution system options such as node and core counts

## Related Github Issues:

- [TUI-246](https://github.com/tapis-project/tapis-ui/issues/246)

## Summary of Changes:

- ExecSystem step supports queue options with validation on a per queue basis

## Testing Steps:

1. Test with FullJobAttrs. 
2. Change exec systems and queues
3. Attempt to change node and core count values beyond parameters of the selected execution systems queue
4. isMPI should allow mutual exclusivity (isMpi means mpiCmd is supported, not isMpi means cmdPrefix is supported)

## UI Photos:

![image](https://user-images.githubusercontent.com/17181582/163221763-1013c06b-29ee-4920-9c04-2ad8471b726b.png)


## Notes:
